internal/fs/inode/file.go

```diff
@@ -673,6 +696,15 @@ func (f *FileInode) CacheEnsureContent(ctx context.Context) (err error) {
 }
 
 func (f *FileInode) CreateEmptyTempFile() (err error) {
+	// Skip creating empty file when streaming writes are enabled
+	if f.local && f.writeConfig.ExperimentalEnableStreamingWrites {
```

Comment:
Should this be for size 0 too?

Reply:
This method is invoked only in CreateFile flow and not open file.
Size 0 files come in open file flow.